packages/blockchain-common/src/registries/FabricGatewayRegistry.ts

```diff
@@ -100,4 +104,8 @@ export class FabricGatewayRegistry extends FileRegistry<FabricGatewayRegistryEnt
         }
     }
 
+    public newFabletEnvironment(name: string, directory: string, url: string): FabletEnvironment {
```

Comment:
I don't think there's any need for this function, might as well just call `new FabletEnvironment(name, directory, url);` inline.